tools/orca/src/model.rs

```diff
@@ -118,6 +121,30 @@ pub async fn person_set_self_mail(
     Ok(parsed_result)
 }
 
+pub async fn person_set_self_password(
+    client: &KanidmClient,
+    person: &Person,
+    pw: &str,
+) -> Result<(TransitionResult, EventRecord), Error> {
+    // Should we measure the time of each call rather than the time with multiple calls?
```

Comment:
We should be using tracing [instrumentation](https://docs.rs/tracing/0.1.38/tracing/attr.instrument.html) then you can get all the various call timings.

Reply:
Actually I'm wondering why we didn't use it in the first place @Firstyear? 🤔

Reply:
Because I didn't know you could use it like this to pull the timings out. 

Lets change it to tracing instrumentation in a follow up PR I think.

Reply:
Makes lots of sense. Will do that once this gets merged ig